go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/forPelevin/gomoji v1.3.1
 	github.com/goccy/go-yaml v1.18.0
 	github.com/google/go-cmp v0.7.0
-	github.com/google/go-github/v75 v75.0.0
+	github.com/google/go-github/v75 v75.0.1-0.20250926113852-138500ee4160
```

Comment:
Not sure if a snapshot would be desirable, but I wanted to give this a try before the next go-github version is out.